deploy/helm/templates/daemonset.yaml

```diff
@@ -26,24 +26,31 @@ spec:
         - name: POD_NAME
           valueFrom:
             fieldRef:
-              apiVersion: v1
-              fieldPath: metadata.namespace
+              fieldPath: metadata.name
         - name: POD_NAMESPACE
           valueFrom:
             fieldRef:
-              apiVersion: v1
               fieldPath: metadata.namespace
         - name: INSTANCE_IP
           valueFrom:
             fieldRef:
-              apiVersion: v1
               fieldPath: status.podIP
         - name: MESH_CONTROLLER
           value: {{ quote .Values.deploy.kmesh.env.meshController }}
         - name: BPF_LOG_SIZE
           value: {{ quote .Values.deploy.kmesh.env.bpfLogSize }}
         - name: KUBERNETES_CLUSTER_DOMAIN
           value: {{ quote .Values.kubernetesClusterDomain }}
+        - name: SERVICE_ACCOUNT
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
```

Comment:
`apiVersion` here could also be deleted?

Reply:
i think so, this is an optional field, and default to `v1`